test-bitcoincore-rpc/src/lib.rs

```diff
@@ -206,6 +215,33 @@ pub trait Api {
   #[rpc(name = "getblock")]
   fn getblock(&self, blockhash: BlockHash, verbosity: u64) -> Result<String, jsonrpc_core::Error>;
 
+  #[rpc(name = "createrawtransaction")]
+  fn create_raw_transaction(
+    &self,
+    utxos: Vec<CreateRawTransactionInput>,
+    outs: HashMap<String, f64>,
```

Comment:
Floats are a bit problematic, but I remember we had difficulties with using `Amount`. I think we should try though, floats are just a bit dangerous, I always worry about rounding.

Reply:
For this to work with `Amount` we need the `Serialize` and `Deserialize` trait on the struct in your fork of the bitcoin rust crate.